modules/ROOT/pages/index.adoc

```diff
@@ -48,11 +48,10 @@ image::os-use-object-store-v2.png[Click Use Object Store V2 in the Runtime Manag
 * The Object Store Connector works for both Object Store version 1 and 2.
 * Object Store v2 enforces a maximum data persistence of 30 days from time of creation.
 * Accounts that have not purchased the Object Store v2 premium SKU are rate limited to 10 transactions per second per application. To go beyond the base 10 TPS rate limit, contact your MuleSoft account team.
-* In Mule 4, Object Store v2 overrides the `_defaultPersistentObjectStore` object store. No further configuration is required.
+* Object Store v2 overrides the `_defaultPersistentObjectStore` object store when creating Mule 4 applications. No further configuration is required.
```

Comment:
We should make clear that this only happens in CH

Reply:
Changed wording to:
`* For cloud applications using Mule 4, Object Store v2 overrides the _defaultPersistentObjectStore object store. No further configuration is required. If a cloud connector or component uses that object store, then you have access to Object Store v2.
+
For cloud applications using Mule 3, Object Store v2 overrides the _defaultUserObjectStore object store. To use Object Store v2 in cloud applications, set the Object Store Reference field to the _defaultUserObjectStore value.`